components/setting_picture.tsx

```diff
@@ -15,35 +14,48 @@ import OverlayTrigger from 'components/overlay_trigger';
 import Tooltip from 'components/tooltip';
 import LoadingWrapper from 'components/widgets/loading/loading_wrapper';
 
-export default class SettingPicture extends Component {
+type Props = {
+    clientError?: ReactNode;
+    serverError?: ReactNode;
+    src?: string | null;
+    defaultImageSrc?: string;
+    file?: File | null;
+    loadingPicture?: boolean;
+    submitActive?: boolean;
+    onRemove?: () => void;
+    onSetDefault?: (() => Promise<void>) | null;
+    onSubmit?: (() => void) | null;
+    title?: string;
+    onFileChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+    updateSection: (e: React.MouseEvent<HTMLButtonElement>) => void;
```

Comment:
When I look at the old prop definition, they are not required. Therefore lets make them optional with `?`

Reply:
Resolved